tests/unit/src/kakarot/accounts/eoa/test_library.py

```diff
@@ -58,6 +67,11 @@ async def test_execute_should_make_all_calls_and_return_concat_results(
                         bytes.fromhex(transaction.get("data", "").replace("0x", ""))
                     ),  # calldata
                 ]
+                # Update ledger
+                account_address = int(transaction["to"], 16)
+                ledger[account_address] = (
+                    int(ledger.get(account_address) or 0) + transaction["value"]
```

Comment:
fyi in python, you can put the default value in the `get` like `ledger.get(account_address, 0)`